parquet-hadoop/src/main/java/parquet/hadoop/InternalParquetRecordReader.java

```diff
@@ -80,6 +81,7 @@
   private long totalCountLoadedSoFar = 0;
 
   private Path file;
+  private CorruptRecordCounter corruptRecordCounter;
```

Comment:
oh, nevermind, I had it as final originally but unfortunately this class is setup by an initialize() method, not the constructor, so this can't be final because it has to get assigned in the initialize() method. An unfortunate consequence of hadoop's pattern of instantiating classes reflectively via their default constructor.